src/main/resources/modules/bronchitis.json

```diff
@@ -23,7 +23,7 @@
           "condition": {
             "condition_type": "Age",
             "operator": "<",
-            "quantity": 18,
+            "quantity": 5,
```

Comment:
Do we want our source patients to be outside of the cohort bounds so we know filtering within the actor job(s) works?

Reply:
agree!